### DISCUSSION TITLE
Question: Does Raytha support "one to many" content type?

### DISCUSSION THREAD

**Discussion by dongleivip** [RaythaHQ/raytha#156] [General]
Hi team, 

I'm seeking a headless CMS to customize it to be a simple _inovice_ tool. I wonder does Raytha support "one to many" content type? Or can advice if is there a sort of work-around to achieve?

**Comment by apexdodge** [comment]
Hi,

We have a field type called `relationship` but it is currently 1-1, not 1-Many. The workaround would be to add a bunch of these fields to the content type and simply call them field 1, field 2, field 3, etc. A table-like/repeater field is something we'd definitely want to add in the future though.
